Sources/Target/Windows/Process.cpp

```diff
@@ -104,10 +122,52 @@ ErrorCode Process::detach() {
   return kSuccess;
 }
 
+BOOL Process::writeDebugBreakCode(uint64_t address) {
+  FARPROC exitThreadAddress =
+      GetProcAddress(GetModuleHandleA("ntdll"), "RtlExitUserThread");
+  if (exitThreadAddress == NULL) {
+    return FALSE;
+  }
+
+  ByteVector codestr(&debugBreakCode[0],
+                     &debugBreakCode[sizeof(debugBreakCode)]);
+  *reinterpret_cast<uint32_t *>(&codestr[kRtlExitUserThreadOffset]) =
+      reinterpret_cast<uint32_t>(exitThreadAddress);
+
+  size_t written;
+  if (writeMemory(Address(address), &codestr[0], codestr.size(), &written) !=
+      kSuccess) {
+    return FALSE;
+  }
+  return TRUE;
+}
+
 ErrorCode Process::interrupt() {
-  BOOL result = DebugBreakProcess(_handle);
-  if (!result)
+  SYSTEM_INFO info;
+  GetSystemInfo(&info);
+
+  uint64_t address;
+  if (allocateMemory(info.dwPageSize, kProtectionExecute | kProtectionWrite,
+                     &address) != kSuccess) {
+    return Platform::TranslateError();
+  }
+
+  if (!writeDebugBreakCode(address)) {
     return Platform::TranslateError();
+  }
+
+  DWORD threadId;
+#if defined(ARCH_ARM)
```

Comment:
Nit: you could do 
```
#if defined(ARCH_ARM)
    // comment...
    address |= 1ULL;
#endif
```

Shorter and reduces the number of unused temporary variables.